README.md

```diff
@@ -1,37 +1,76 @@
-                                                                                
-                                                                                
-                                                                                
-                                                                                
-           .&,%&%%*@*                                                           
-         .%%#%%&%&%%&%                                                          
-          %%&(......,/                                                          
-          %%%&&%%*.....,(                                                       
-          /@@@@&%,.......,*                                                     
-           *&%@@#,.........,,.                                                  
-            ##((/,,..........,,,((                                              
-                **,,...........#%%%%%&&&&&&&%%%%%%%%#*.. ..,,**                 
-                 **,,,,,....  .###%%%%%%%%%%%%%##(...*%%%%(. .,*                
-                 //**,,,,...   .*#####%%%%%%%##(,..,.*##%%%%%%,,.               
-                  &&#(//*,......,*%##%%%%%*     ..,,,..,%%%%%&/,,               
-                   &&&%/*,,%&&#,,*%%%%%(..    ..,,,,,... (%%/,**,               
-                    &&&(/*/&&&&/,**/(/,......,,,,***,.......,,**.               
-                     ,&&(/*,#&%,,,****,,,,,*****///**,....,,,**,                
-                       %&#(/*,,,,*/////////////((##//**,,,,,***,                
-                         ,//*,.,/(((/(((((((((((((((#(/****,,/(/.               
-                           **,,*(#((/        ,((((((((#(/**,,,#%#               
-                           ,*. ./((((.         *###((####(*,,,/%%*              
-                            *,.,.////(          %% ###(### /***#%%,             
-                            ,..,* //*//              ////  (*,, #%,             
-                           *,.., /////(            (/////  *,..*(.              
-                         (#(/** %%##((            %%#((*  (/**.                 
-                                                          .,*                   
-                                                                                
-                                                                                
-                                                                                
-                                                                                
+You can add new Time Entries and Team Members with some limitations. You can filter these entries and members by some primary variables of these objects.
+
+## Installation
+
+To get a demo of this front end web app, check out this repo and use
+
+```bash
+npm run dev
+```
+
+### Stack
+
+HTML, CSS, JS ...
+Made with React, styled-components, Zustand, StoryBook, Jest & Cypress
```

Comment:
No need to mention HTML, CSS and JS. React can be replaced with Next.js, as it's a more pronounced part of the app. Finally, if you mention your used stack, it's nice to add url's so people can check it out.